src/js/actions/tools.js

```diff
@@ -878,7 +879,8 @@ define(function (require, exports) {
                 specs.push({
                     key: utilShortcuts.GLOBAL.TOOLS.SHAPE,
                     modifiers: {},
-                    fn: activateTool
+                    fn: activateTool,
+                    name: "Other " + tool.name.toLowerCase() + " shortcut"
```

Comment:
I wasn't sure about this but I agree that there should be one rectangle shortcut only. I'll make the shortcut "R" the primary one for now.